### ISSUE TITLE
Optimize usability for laptop and mobile

### ISSUE THREAD

**Issue by Dynastybball** [Dynastybball/DynastyBball#5]
One of the most important improvements your version of the tool provides compared to mine is that it should be far better for mobile devices. Optimizing it for use would be fantastic.

**Comment by e0** [comment]
I feel like this should be the one we want to tackle next. It is doable without any additional data and would improve the user experience a lot. By the way, have you taken down the GitHub page for the project? I visited https://dynastybball.github.io/dynastybball but it didn't seem to work. It would make it easier to test on different devices if it was working.

**Comment by Dynastybball** [comment]
I agree. I didn't take it down on purpose...let me see if I can't put it back up.

**Comment by Dynastybball** [comment]
I believe I found the issue: apparently capitalization matters. Try: 

https://dynastybball.github.io/DynastyBball/

**Comment by e0** [comment]
Ah thank you that worked for me! In terms of usability, is there anything specific that you want to focus on?

**Comment by Dynastybball** [comment]
My sense is that usability on mobile is a big issue. Building out the tool so that it can be used on a phone would be a big benefit.
